core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -2315,8 +2379,11 @@ struct crypto_ops crypto_ops = {
 		.copy = copy,
 		.free = bn_free,
 		.clear = bn_clear
-	}
+	},
 #endif /* _CFG_CRYPTO_WITH_ACIPHER */
+	.prng = {
+		.read_data = prng_read_data,
```

Comment:
I find `_data` superfluous, `.read` is enough (and is the name used in LTC).
I would also export `add_entropy` and maybe other functions, and call `add_entropy` in some select locations of TEE core, but that is better left to another patch I guess.